products/BellHybrid/apps/application-bell-meditation-timer/windows/StatisticsWindow.cpp

```diff
@@ -5,24 +5,32 @@
 
 #include "MeditationMainWindow.hpp"
 
-#include <ListView.hpp>
-#include <common/data/StyleCommon.hpp>
-#include <apps-common/ApplicationCommon.hpp>
 #include <module-gui/gui/input/InputEvent.hpp>
-#include <apps-common/InternalModel.hpp>
-
-namespace
-{
-    constexpr auto height     = 400;
-    constexpr auto width      = 380;
-    constexpr auto top_margin = 41;
-} // namespace
+#include <module-gui/gui/widgets/SideListView.hpp>
 
 namespace app::meditation
 {
     using namespace gui;
+
+    /// The statistics list items are handled a bit differently than the rest. Instead of navigating using
+    /// a light press/back combo, they use knob rotate events.
+    InputEvent transformInputEvent(const InputEvent &inputEvent)
```

Comment:
IMO would be better to change function name to something like knobBasedTransformInputEvent or transformInputEventknobBased(or similar) and delete commets